webknossos/webknossos/annotation/annotation.py

```diff
@@ -124,6 +146,63 @@ def download(cls, annotation_path: str) -> "Annotation":
 
         return download_annotation(annotation_type, annotation_id)
 
+    def _skeleton_zip_path(self) -> Union[Openable, PathLike, str]:
+        nml_files = [i for i in self._filelist if i.endswith(".nml")]
+        assert len(nml_files) == 1
+        return _ZipPath(self._zipfile, nml_files[0])
+
+    @contextmanager
+    def _open_nml(self) -> Iterator[IO[bytes]]:
+        """
+        This method can be used to open the annotation's inner NML
+        file directly.
+        """
+        file_path = self._skeleton_zip_path()
+
+        if isinstance(file_path, Openable):
+            with file_path.open(mode="rb") as file_handle:
+                yield file_handle
+        else:
+            with open(file_path, "rb") as file_handle:
+                yield file_handle
+
+    def save_to_file(self, path: Union[Path, str]) -> None:
```

Comment:
I'd rather call this `save` according to our method-naming discussion. Please also add a link to this in the table in the top-level `__init__.py`, so it appears on
https://docs.webknossos.org/api/webknossos.html